zerver/lib/validator.py

```diff
@@ -144,3 +148,10 @@ def f(var_name, val):
                      'value': val})
         return None
     return f
+
+def validate_login_email(email):
+    # type: (Text) -> None
+    try:
+        validate_email(email)
+    except ValidationError as err:
+        raise JsonableError(str(err.message))
```

Comment:
We should add a simple test for this to `ValidatorTest`, to preserve it having 100% test coverage for this file.